### PR TITLE
use separate var to register output of each health check task

### PR DESCRIPTION
**Pull request by utkarsh5474**
# Description

Our code fell prey to an infamous ansible default behaviour - registering vars even for skipped tasks. 
Refer https://github.com/ansible/ansible/issues/17500 or https://github.com/ansible/ansible/issues/4297
The suggested solution is to use separate variables to register output of separate tasks. 

Fixes # https://github.com/confluentinc/cp-ansible/issues/954 and https://github.com/confluentinc/cp-ansible/issues/1041

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Tested locally with the fix. 

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: roles/confluent.kafka_broker/tasks/health_check.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@
   shell: |
     {{ binary_base_path }}/bin/kafka-topics --bootstrap-server {{inventory_hostname}}:{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['port']}} \
       --describe --under-replicated-partitions --command-config {{kafka_broker.client_config_file}}
-  register: topics
+  register: topics1
```
Comment (erikgb) [inline roles/confluent.kafka_broker/tasks/health_check.yml:6]: Maybe try to name these variables better?

Comment (utkarsh5474) [reply roles/confluent.kafka_broker/tasks/health_check.yml:6]: Sure Erik. I am thinking
topic1 -> urp_topic
topic2 -> urp_topic_secrets_protection
WDYT, any names you'd like to suggest?

Comment (erikgb) [reply roles/confluent.kafka_broker/tasks/health_check.yml:6]: SGTM! 👍

**Review by erikgb — APPROVED** [review]
LGTM! Thanks for fixing this nasty bug!

**Review by nsharma-git — APPROVED** [review]

**Pull request merged**